examples/standalone/gtest_setup/CMakeLists.txt

```diff
@@ -1,36 +1,33 @@
-cmake_minimum_required(VERSION 3.11.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
```

Comment:
Any reason for changing this?

Reply:
On 3.11:
```
ryan@B650-970:~/Dev/gz_harmonic_ws/src/gz-sim/examples/standalone/gtest_setup$ cmake -S . -B build
CMake Error: Could not find CMAKE_ROOT !!!
CMake has most likely not been installed correctly.
Modules directory not found in
/home/ryan/Dev/cmake_ws/cmake311/cmake/build/share/cmake-3.11
CMake Error: Error executing cmake::LoadCache(). Aborting.
```

On 3.18 which I have easily, it builds per the README.

Reply:
Even though the root level CMakeLists file supports 3.10, I think it's safe to change the examples and I think this is a good enough reason.